executor/linux/build.go

```diff
@@ -217,11 +204,84 @@ func (c *client) PlanBuild(ctx context.Context) error {
 		return fmt.Errorf("unable to inspect volume: %w", err)
 	}
 
-	// update the init log with volume info
+	// update the init log with network info
```

Comment:
Are we sure we want to change this here? 👍

I ask because I think this line is adding `volume` information.